TranslateAddressProviderTest failure [API-1111]

On windows and cd6240ed9569b6f8fdb9ef7339ea7e6e3beb6448
```
 1) TranslateAddressProviderTest
       should provide true for default config and publicly reachable single member:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (test\unit\network\TranslateAddressProviderTest.js:236:37)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  2) TranslateAddressProviderTest
       should provide true for default config and publicly reachable members:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (test\unit\network\TranslateAddressProviderTest.js:251:37)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

--- a/test/unit/network/TranslateAddressProviderTest.js
+++ b/test/unit/network/TranslateAddressProviderTest.js
@@ -33,7 +33,8 @@ const { UuidUtil } = require('../../../lib/util/UuidUtil');
 describe('TranslateAddressProviderTest', function () {
     const PROPERTY_DISCOVERY_PUBLIC_IP_ENABLED = 'hazelcast.discovery.public.ip.enabled';
     const REACHABLE_HOST = '127.0.0.1';
-    const UNREACHABLE_HOST = '192.168.0.1';
+    // Test network ip, should be unreachable
+    const UNREACHABLE_HOST = '198.51.100.1';
 
     let loggerStub;
     let fakeMemberServer;

--- a/test/unit/network/TranslateAddressProviderTest.js
+++ b/test/unit/network/TranslateAddressProviderTest.js
@@ -33,7 +33,7 @@ const { UuidUtil } = require('../../../lib/util/UuidUtil');
 describe('TranslateAddressProviderTest', function () {
     const PROPERTY_DISCOVERY_PUBLIC_IP_ENABLED = 'hazelcast.discovery.public.ip.enabled';
     const REACHABLE_HOST = '127.0.0.1';
-    // Test network ip, should be unreachable
+    // 198.51.100.0/24 is assigned as TEST-NET-2 and should be unreachable: https://en.wikipedia.org/wiki/Reserved_IP_addresses
     const UNREACHABLE_HOST = '198.51.100.1';
 
     let loggerStub;

